package/Runtime/Scripts/Internal/NimbusAdUnit.cs

```diff
@@ -52,7 +52,7 @@ public sealed class NimbusAdUnit {
 			_androidController = null;
 			_androidHelper = null;
 # elif UNITY_IOS
-			OnDestroyIOSAd();
+			DestroyIOSAd();
```

Comment:
If this is an event it should be named OnDestroyIOSAd

Reply:
https://github.com/timehop/nimbus-unity/pull/21/commits/13470b73968ffa02a341f14c6e7a7a8285630bd5